editions/tw5.com/tiddlers/wikitext/Wikitext Cheat-Sheet.tid

```diff
@@ -0,0 +1,143 @@
+created: 20220821184022101
+modified: 20221010130927409
+tags: WikiText
+title: Wikitext Cheat-Sheet
+type: text/vnd.tiddlywiki
+
+\define extLink(text tid) [ext[$text$|https://tiddlywiki.com/#$tid$]]
```

Comment:
Why is the cheat sheet so special that it needs it's own export button? We really need to strive for consistency.

Surely it would make more sense to just arrange a tag pill that can be dragged to export the cheat sheet to another wiki?

Reply:
>Why is the cheat sheet so special that it needs it's own export button?

It should be easy for users to export the basic stuff and **extend the info with their own docs**. ... It's an attempt to get more feedback and it may even be possible that someone sends us an improved JSON file that we could include into the docs, without the need, that we need to do the work.

It would be possible to make the cheat-sheet a plugin. -- but users won't improve our plugins, they just complain that info is missing.